docs/i18n/fr/docusaurus-plugin-content-docs/current/getting-started/demo/with-docker.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 2
+---
+
+# Via Docker
+
+```mdx-code-block
+import TerminalWindow from '@site/src/components/TerminalWindow';
+```
+
+> Ce document détaille les étapes nécessaires pour faire tourner une Hydra Head dans un environment de démo, avec Docker.
+
+La démo comprend:
+
+- Un cluster de trois noeuds Hydra, connectés point-à-point, et ayant chacun une identité (`alice`, `bob`, `carol`) associée à une paire de clés publique/privée;
```

Comment:
typo: lists in French are not capitalised
suggestion: add an extra `&nbsp;` before the terminating semicolon